rero_ils/modules/notifications/dispatcher.py

```diff
@@ -0,0 +1,66 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of RERO ILS.
+# Copyright (C) 2019 RERO.
+#
+# RERO ILS is free software; you can redistribute it
+# and/or modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# RERO ILS is distributed in the hope that it will be
+# useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with RERO ILS; if not, write to the
+# Free Software Foundation, Inc., 59 Temple Place, Suite 330, Boston,
+# MA 02111-1307, USA.
+#
+# In applying this license, RERO does not
+# waive the privileges and immunities granted to it by virtue of its status
+# as an Intergovernmental Organization or submit itself to any jurisdiction.
+
+"""API for dispatch Notifications."""
+
+from __future__ import absolute_import, print_function
+
+from flask_security.utils import config_value
+from invenio_mail.api import TemplatedMessage
+from invenio_mail.tasks import send_email
+
+
+class Dispatcher():
+    """Dispatcher notifications class."""
+
+    def send(self, notification=None):
+        """Send dispatch."""
```

Comment:
change it with something like: Send all notifications or Dispatch all notifications, etc.

Reply:
Changed to: `dispatch_notification`